src/test/zerocoin_encryption_tests.cpp

```diff
@@ -0,0 +1,156 @@
+//
+// Copyright (c) 2017 The PIVX developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "../libzerocoin/Accumulator.h"
+#include "../uint256.h"
+#include "../primitives/zerocoin.h"
+#include "../sync.h"
+#include "../main.h"
+#include "../crypter.h"
+#include "../wallet.h"
```

Comment:
not digging the relative includes here

Reply:
Ahhh, nice catch. I have just recently configured CMakeLists.txt in CLion to work off of absolute paths. Forgot to actually crawl through and change all the relatives :+1: